Assets/Scripts/Controllers/MouseController.cs

```diff
@@ -369,6 +396,26 @@ void UpdateDragging()
                             {
                                 isValid = true;
                             } 
+                        } else if (dragType == "path")
```

Comment:
The "else if (...)" should go on a new line.

Maybe we should use an enum for the drag types.